generate_script.sh

```
@@ -1,5 +1,6 @@
-IOC_PATH="$(realpath test439/test439.ioc)"
-PROJECT_PATH="$(realpath test439)"
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)